org.mozilla.Thunderbird.json

```diff
@@ -0,0 +1,121 @@
+            ]
+        },
+	{
+	    "name": "icu",
+        "cleanup": ["/bin/*", "/sbin/*"],
+	    "sources": [
+	       {
+		    "type": "archive",
+	            "url":"http://download.icu-project.org/files/icu4c/60.1/icu4c-60_1-src.tgz",
+	            "sha256":"f8f5a6c8fbf32c015a467972bdb1477dc5f5d5dfea908b6ed218715eeb5ee225"
+	       }
+	    ],
+	   "subdir": "source"
+	},
+        {
+            "name": "thunderbird",
+            "buildsystem": "simple",
+            "build-options":{
+                "cflags": "-fno-delete-null-pointer-checks -fno-lifetime-dse -fno-schedule-insns2",
+                "cxxflags": "-fno-delete-null-pointer-checks -fno-lifetime-dse -fno-schedule-insns2",
+                "env":{
+                    "VERSION": "52.4.0"
+                }
+            },
+            "build-commands": [
+                "make -f client.mk",
+                "make -f client.mk install INSTALL_SDK=",
+                "mkdir -p /app/share/icons/hicolor/16x16/apps",
+                "cp /app/lib/thunderbird-${VERSION}/chrome/icons/default/default16.png /app/share/icons/hicolor/16x16/apps/thunderbird.png",
+                "mkdir -p /app/share/icons/hicolor/22x22/apps",
+                "cp /app/lib/thunderbird-${VERSION}/chrome/icons/default/default22.png /app/share/icons/hicolor/22x22/apps/thunderbird.png",
+                "mkdir -p /app/share/icons/hicolor/24x24/apps",
+                "cp /app/lib/thunderbird-${VERSION}/chrome/icons/default/default24.png /app/share/icons/hicolor/24x24/apps/thunderbird.png",
+                "mkdir -p /app/share/icons/hicolor/32x32/apps",
+                "cp /app/lib/thunderbird-${VERSION}/chrome/icons/default/default32.png /app/share/icons/hicolor/32x32/apps/thunderbird.png",
+                "mkdir -p /app/share/icons/hicolor/48x48/apps",
+                "cp /app/lib/thunderbird-${VERSION}/chrome/icons/default/default48.png /app/share/icons/hicolor/48x48/apps/thunderbird.png",
+                "mkdir -p /app/share/icons/hicolor/256x256/apps ",
+                "cp /app/lib/thunderbird-${VERSION}/chrome/icons/default/default256.png /app/share/icons/hicolor/256x256/apps/thunderbird.png",
+                "mkdir -p /app/share/applications",
+                "cp org.mozilla.Thunderbird.desktop /app/share/applications",
+		"mkdir -p /app/share/appdata",
+		"cp org.mozilla.Thunderbird.appdata.xml /app/share/appdata"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "mozconfig"
+                },
+                {
+                    "type": "file",
+                    "path": "org.mozilla.Thunderbird.desktop"
+                },
+		{
+			"type": "file",
+			"path": "org.mozilla.Thunderbird.appdata.xml"
+		},
+                {
+                    "type": "archive",
+                    "url": "https://ftp.mozilla.org/pub/thunderbird/releases/52.4.0/source/thunderbird-52.4.0.source.tar.xz",
```

Comment:
52.5.0 has been released

Reply:
Thanks, I was aware of this and will make changes. I'll need to put in a new `release` tag in the appstream data correct?